Account/FirefoxAccount.swift

```diff
@@ -152,7 +152,7 @@ public class FirefoxAccount {
         return nil
     }
 
-    public enum AccountError: Printable, ErrorType {
+    public enum AccountError: CustomStringConvertible, MaybeErrorType {
```

Comment:
I don't think we need to inherit CustomStringConvertible since MaybeErrorType already inherits that type